### PR TITLE
osu-lazer: 2021.703.0 -> 2021.707.0

### PR DESCRIPTION
**Pull request by sefidel**
###### Motivation for this change
Bump versions

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
@ofborg eval

File: pkgs/games/osu-lazer/default.nix [hunk 1]
```diff
@@ -16,13 +16,13 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "osu-lazer";
-  version = "2021.703.0";
+  version = "2021.707.0-lazer";
```
Comment (oxalica) [inline pkgs/games/osu-lazer/default.nix:19]:
```suggestion
  version = "2021.707.0";
```

It tagged both `2021.707.0` and `2021.707.0-lazer` to the same commit. I'm not sure why there are two but we should not include this suffix in `version`.

**Review by oxalica — CHANGES_REQUESTED** [review]

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 130283` at 8a707c19 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>osu-lazer</li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/fixup-phase.md">fixup-phase</a>

`fixupPhase` should not be overridden, use `postFixup` instead.

Near pkgs/games/osu-lazer/default.nix:103:3:
```
    |
103 |   fixupPhase = ''
    |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `alsa-plugins`.
Near pkgs/games/osu-lazer/default.nix:3:49:
```
  |
3 | , ffmpeg_4, alsa-lib, SDL2, lttng-ust, numactl, alsa-plugins
  |                                                 ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Consider adding a comment explaining the purpose of this patch on the line preceeding.
Near pkgs/games/osu-lazer/default.nix:28:15:
```
   |
28 |   patches = [ ./bypass-tamper-detection.patch ];
   |               ^
```
</li>
  </ul>
</details>

File: pkgs/games/osu-lazer/default.nix (hunk 1 above)
Comment (boppyt) [reply pkgs/games/osu-lazer/default.nix]: Yeah, I was confused by that too. Since they're same commit, I think we should do that.

**Review by fufexan — APPROVED** [review]

**Pull request merged**